### DISCUSSION TITLE
Feature: CDU space and drawing at the sides of Rack?

### DISCUSSION THREAD

**Discussion by LFrank2021** [opendcim/openDCIM#1508] [Ideas]
Hi,

I have not found a way to have CDU at the sides of the Rack.
At least Rittal (which we use) has this feature.
Is there already a way to define CDU at the sides and get openDCIM to draw them?
Or is it possible to get that in a future release?

Currently I have defined my CDU at height=0 on the backside and without any position.
I assume this feature would have to be flexible (at least) 1-4 side slots for CDU, maybe two on each side, maximum height = Rack height.

Kind regards
Frank

**Comment by alex001x** [comment]
Good morning,
Currently Opendcim allows you to display the PDUs in U0 in a "Power distribution" frame on "cabnavigator.php" but does not draw the PDU outside the rack or then goes to Umax +1 or U43 if max =42, the device in U0 are counted in the reports. Also visible in U43 in rowview.php
Alex.
